build/stf-run-ci/tasks/setup_stf.yml

```diff
@@ -49,6 +49,14 @@
         sourceNamespace: openshift-marketplace
   when: service_telemetry_operator_subscription_manifest is not defined
 
+# enable catalogsource
```

Comment:
Seemed weird to subscribe before creating the catalogsource, so I just moved this up in the task list.